apps/marsha/vars/vault/main.yml.j2

```diff
@@ -12,19 +12,25 @@ DJANGO_SECRET_KEY: {{ lookup('password', '/dev/null length=30') }}
 # DJANGO_SENTRY_DSN: https://super:marsha@sentry.io/foo
 
 # FIXME uncomment this block and replace with your AWS credentials.
-# You can refer to the marsha documentation to know what are this variables: 
+# You can refer to the marsha documentation to know what are this variables:
```

Comment:
```suggestion
# You can refer to Marsha's documentation to know what are these variables:
```

Reply:
This our of the scope of this PR :trollface:

Reply:
Yes, I should have seen this earlier :stuck_out_tongue: